src/Nethermind/Nethermind.Consensus.Clique/ICliqueModule.cs

```diff
@@ -24,15 +24,34 @@ namespace Nethermind.Consensus.Clique
     [RpcModule(ModuleType.Clique)]
     public interface ICliqueModule : IModule
     {
+        [JsonRpcMethod(Description = "Retrieves a snapshot of all clique state at a given block.", Returns = "Object", IsImplemented = true)]
         ResultWrapper<Snapshot> clique_getSnapshot();
+
+        [JsonRpcMethod(Description = "Retrieves the state snapshot at a given block.", Returns = "Object", IsImplemented = true)]
```

Comment:
Similarly as above - unfortunately state snapshot has a different meaning in Ethereum. State is a collection of accounts and storage values. Snapshot is part of the consensus process with a list of validators and their votes.